addons/payment_razorpay/const.py

```diff
@@ -98,12 +98,15 @@
     'ZAR',
 ]
 
-# The codes of the payment methods to activate when Razorpay is activated.
-DEFAULT_PAYMENT_METHOD_CODES = {
-    # Primary payment methods.
+# The codes of primary payment methods which directly supported by orders API.
+PRIMARY_PAYMENT_METHOD_CODES = {
```

Comment:
I find it wrong to name a variable `PRIMARY_PM_...` if it does not contain all supported primary PMs. A primary PM is simply a PM that has no "parent", i.e., that is not a brand PM.

Instead, we could keep the `DEFAULT_PAYMENT_METHOD_CODES` variable as-is and simply add a new `FALLBACK_PAYMENT_METHOD_CODES` variable, for example, and list payment methods that require a fallback.

Reply:
Ohh oops :cry: 
Again I thought I can use existing things :sweat: 
Done :innocent: